pkg/client/redhatsso/client.go

```diff
@@ -131,7 +129,7 @@ func (c *rhSSOClient) GetToken() (string, error) {
 	if err != nil {
 		return "", err
 	}
-
+	c.cache.Set(cachedTokenKey, tokenData.AccessToken, cacheCleanupInterval)
```

Comment:
The cache clean up interval and expire time are set to 5 minutes: 
https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/f1c209590ac6662d1da44f1548d6c5aa8fac18de/pkg/client/redhatsso/client.go#L22-L23
I think this corresponds to the token expiration time, would it make sense to reduce the interval to `4 minutes and 59 seconds` so that we evict the entry from the cache before the actual token expiration time - to avoid cases where we use a token that's going to expire in the next second or so?